src/lib/learn-client/api/progress/get-all-progress.ts

```diff
@@ -31,9 +31,23 @@ const getAllProgress = async ({
 	tutorialIds,
 	collectionIds,
 }: GetAllProgressOptions): Promise<GetAllProgressResult> => {
+	/**
+	 * TODO: figure out why this happens sometimes!
+	 * Maybe related to "queryClient.invalidate"? That's my best guess so far.
+	 */
+	const doesNotHaveAccessToken = typeof accessToken == 'undefined'
```

Comment:
🚧 Need to figure this out, very open to ideas if anyone has any. Mostly focused on getting the core "batching" part of things sorted first though.